shadowsocks-csharp/View/LogForm.cs

```diff
@@ -148,12 +148,12 @@ private void UpdateTexts()
             FileMenuItem.Text = I18N.GetString("&File");
             OpenLocationMenuItem.Text = I18N.GetString("&Open Location");
             ExitMenuItem.Text = I18N.GetString("E&xit");
-            CleanLogsButton.Text = I18N.GetString("&Clean Logs");
+            CleanLogsButton.Text = I18N.GetString("&Clear Logs");

```

Comment:
Please change the button name as well.

Reply:
My programming is so bad, after the change button name, the building has failed.  :'(